<launch>
  <include file="$(find sim2real_dynamics_sawyer)/launch/slide_control/slide_object_obs.launch"/>

   <node name="slide_control" pkg="sim2real_dynamics_sawyer" type="slide_control.py" output="screen" >
    </node>
</launch>